workspaces/src/worker/impls.rs

```diff
@@ -186,6 +187,21 @@ impl Worker<Sandbox> {
         self.workspace.patch_state(contract_id, key, value).await
     }
 
+    /// Patch account into the sandbox network, given an optional Account fields. This will allow
+    /// us to set amount or locked amount of NEAR tokens, change code hash, and storage usage.
+    pub async fn patch_account(
+        &self,
+        account_id: &AccountId,
+        amount: Option<Balance>,
+        locked: Option<Balance>,
+        code_hash: Option<CryptoHash>,
+        storage_usage: Option<StorageUsage>,
```

Comment:
> Not sure, how to make a friendly API for this. Thinking of using builder here.

yeah definitely, would be great if we can get a builder going for this. You can take a look at [patch.rs](https://github.com/near/workspaces-rs/blob/main/workspaces/src/rpc/patch.rs) to see how to get one going. Probably also best to put it there as well